src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -298,6 +298,8 @@ class GcsServer {
   std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage_;
   /// Stores references to URIs stored by the GCS for runtime envs.
   std::unique_ptr<ray::RuntimeEnvManager> runtime_env_manager_;
+  /// Local task manager.
+  std::unique_ptr<raylet::ILocalTaskManager> local_task_manager_;
```

Comment:
In GCS, it should always be NoopLocalTaskManager

Reply:
I did initialize the task manager as no-op manager.
https://github.com/ray-project/ray/blob/2688cdefe8d75dc8735eb95bf03b1140af314716/src/ray/gcs/gcs_server/gcs_server.cc#L397-L400

Reply:
oh, I mean we should change to `std::unique_ptr< NoopLocalTaskManager > local_task_manager_;` to make it clear?

Reply:
OK, I will change it; but generally I prefer to use unique pointer of base class, since it allows us to do mock-based injection better.

Reply:
Updated.